Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1578,8 +1578,9 @@ void MainWindow::on_actionLoad_plugin_triggered()
     Q_FOREACH(QString fileName, paths)
     {
       if(fileName.contains("plugin") && QLibrary::isLibrary(fileName)) {
+          QFileInfo fileinfo(fileName);
```

Comment:
Be careful with the indentation. Maybe your editor is not configured properly.
  - The indentation width is 2.
  - The TAB width is 8.
  - We should indent preferably with spaces instead of TAB (that is: the <kbd>tab</kbd> key should insert spaces).